experiments/ClimaEarth/run_cloudless_aquaplanet.jl

```diff
@@ -310,12 +311,15 @@ function solve_coupler!(cs)
     ## step in time
     for t in ((tspan[begin] + Δt_cpl):Δt_cpl:tspan[end])
         # Update date
-        cs.dates.date[] = TimeManager.current_date(cs, t)
+        cs.dates.date[] = TimeManager.current_date(cs, cs.t[])
```

Comment:
`cs` already contains `t`, so we probably don't need to pass both `cs` and `cs.t[]`